packages/block-library/src/post-excerpt/edit.js

```diff
@@ -122,15 +150,14 @@ export default function PostExcerptEditor( {
 	/**
 	 * The excerpt length setting needs to be applied to both
 	 * the raw and the rendered excerpt depending on which is being used.
+	 * If there is no excerpt the value is set to '' to make sure it is not undefined (See line 89).
 	 */
-	const rawOrRenderedExcerpt = !! renderedExcerpt
-		? strippedRenderedExcerpt
-		: rawExcerpt;
+	let rawOrRenderedExcerpt = rawExcerpt || strippedRenderedExcerpt;
```

Comment:
Which excerpt variation is preferred here when we have both?

Reply:
Hmm yes In this case the `RawExcerpt` should be prioritized because it is the customized excerpt text.

Reply:
I think the previous logic was wrong and this would correct it.
Looking at the code **before** #44964, the `rawExcerpt` comes first.